tidyblocks/tidyblocks.js

```diff
@@ -169,7 +169,7 @@ const tbMedian = (values) => {
 }
 
 /**
- * Find median value.
+ * Find minimum value.
```

Comment:
Whoops - why is this in this PR? (It's a good fix, it just doesn't belong in the same branch as the user guide changes.)